text/0000-intro-video-script.md

```diff
@@ -0,0 +1,73 @@
+# Meta
+[meta]: #meta
+- Name: Intro video script
+- Start Date: 2021-05-18
+- Author(s): yaelharel
+- RFC Pull Request: (leave blank)
+- CNB Pull Request: (leave blank)
+- CNB Issue: (leave blank)
+- Supersedes: N/A
+
+# Summary
+[summary]: #summary
+
+Improving documentation is a project priority for 2021.
+This RFC is an attempt to create an easy and clear starting point for new users and contributors.
+
+# Definitions
+[definitions]: #definitions
+
+N/A
+
+# Motivation
+[motivation]: #motivation
+
+It is currently relatively challenging for new users and contributors to understand what buildpacks are and what is the purpose of our project.
+It can be very useful to have a ~1 minute introductory video to provide high-level answers to these questions.
+We can post this video on our website and share it on social media.
+Hopefully this will help familiarize more people with our project.
+
+
+# What it is
+[what-it-is]: #what-it-is
+
+### The script
+
+When was the last time you struggled to wrangle yet another Dockerfile? As an application developer, you should only have to focus on what you do best - writing code, without having to worry about your container build strategy. Buildpacks can help you with just that! They are a better approach to building container images for applications.
+
+Buildpacks are programs that transform source code into container images by analyzing the code and determining the best way to build it. They can provide dependencies, configuration, run compilation, and much more! A buildpack can be written by anyone, and shared with everyone! Google, Heroku, Salesforce, and the Paketo project provide buildpacks for a wide variety of language families, and you can discover even more by searching the buildpack registry. With buildpacks, you can use best-in-class container build strategies from industry experts with ease.
+
+Even more so, the Cloud Native Buildpacks project provides a specification for buildpack authors and platforms that ensures builds are fast, secure, customizable, and can run on any cloud. It is a Cloud Native Computing Foundation project with many existing integrations and many more to come.
```

Comment:
I think it'd be good to add back in some of the copy for social proof in this paragraph:

> Once applications are in containers there is an incredible community of tools and products that can help get your app online and support its growth. Heroku, Hashicorp Waypoint, VMWare Tanzu, Spaceship, CircleCI, and Google Cloud Run all support using buildpacks to make container images.

Reply:
Thank you for your comment!
As I wrote https://github.com/buildpacks/rfcs/pull/161/files#r637173592, my top concern is that it'll make our script too long.
We're trying to create a very basic 1 minute video and I'm afraid that even before the additions that you and Sam suggested, our script is too long.

Reply:
My 2c: One thing to keep in mind is that this script will be subjective to additional changes throughout the next steps of productions. I wouldn't be too hung up on specific word counts. If the additions add value they may be worth a few more seconds of air time. It's probably better to have too much to cut out in the next step as oppose to not enough material.

Reply:
@thephw, I took @jromero advice and added your sentence as is for now.